### PR TITLE
Update password reset to add email

### PR DESCRIPTION
**Pull request by joshsmith**
# What's in this PR?

- Adds email template for forgot password
- Fixes a `Poison.EncodeError` for a mistaken change in `CodeCorps.Cloudex.CloudinaryUrlTest`; introduces `placehold.it` URLs here that are testable
- Minor fix for password reset view
- Copy changes
- Changes reset expiration from 14 days to 1 hour

### CODE REVIEW

**Comment by joshsmith** [comment]
Looked at coverage and no meaningful reason for drop other than known `schema` issues with `excoveralls`.

File: web/views/password_reset_view.ex [hunk 1]
```diff
@@ -1,9 +1,11 @@
 defmodule CodeCorps.PasswordResetView do
   use CodeCorps.Web, :view
 
-  def render("show.json", %{token: token}) do
+  def render("show.json", %{email: email, token: token, user_id: user_id}) do
     %{
-      token: token
+      email: email,
+      token: token,
```
Comment (snewcomer) [inline web/views/password_reset_view.ex:7]: Not seeing the token/user_id being consumed on the [frontend](https://github.com/code-corps/code-corps-ember/blob/84d91618ccb464b112555298600b366aeb53dc7f/app/routes/password/reset.js#L30).

File: emails/forgot_password.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```
Comment (snewcomer) [inline emails/forgot_password.html:1]: What is the need for this file if there is a postmark template?

Comment (joshsmith) [reply emails/forgot_password.html:1]: @snewcomer for us to be able to design and preview it locally.

File: web/views/password_reset_view.ex (hunk 1 above)
Comment (joshsmith) [reply web/views/password_reset_view.ex:7]: `user_id` is used by ESA, and we might as well have the token, I think.

**Pull request merged**